povm_toolbox/quantum_info/product_povm.py

```diff
@@ -54,6 +77,9 @@ def get_prob(
     ) -> float | dict[tuple[int, ...], float] | np.ndarray:
         """Return the outcome probabilities given a state rho.
 
+        .. note::
+           TODO: align this docstring with that of :class:`.BasePOVM`.
```

Comment:
This docstring should be possible to be removed once aligned with the base class (see also previous comment).